### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by miguelglezb**
What's new:

- Getting started contains a quickstart, as well as detailed instructions for installation of PyKOALA and Python environments.
- Developer guide (empty) with a mention on forking the repo for contributions.
- Updated list of requirement list
- Remove unnecessary directories from previous MR (lib64, build and bin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,19 +40,14 @@ PyKOALA has the following pre-requisites
...
 
 ```
 git clone https://github.com/pykoala/koala.git
 ```
 
-Second part is installing it in your local machine with pip:
-
-```
-cd koala ; pip install . 
-```
```
Comment (PabloCorcho) [inline README.md:55]: I guess including a brief statement on how to install the package (i.e. `cd /repository; python3 -m pip install .`) would help users with little experience on python.

**Comment by PabloCorcho** [comment]
Apart from my minor comment, I think this is ready to be merged.

File: README.md (hunk 1 above)
Comment (paranoya) [reply README.md:55]: I agree

File: docs/developer-guide/fork.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Forking PyKOALA
+===============
+
+Working on a fork enables you to develop your own copy of PyKOALA, contributing your code back to the main repository through a "pull request."
+
+How to fork
+-----------
+First, click “Fork this repository” on the main PyKOALA repository in github. You then have your own copy of Phantom in github.com/USERNAME/koala
```
Comment (paranoya) [inline docs/developer-guide/fork.rst:8]: Phantom?

File: docs/getting-started/get-copy.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+To get PyKOALA, clone the repository, using git:
+
+::
+    
+    git clone https://github.com/pykoala/koala.git
+
+
+Alternatively, you can create your own :doc:`fork<../developer-guide/fork>` if you want to contribute to the code. We describe the steps for developers in this :doc:`guide<../developer-guide/index>`.
+
```
Comment (paranoya) [inline docs/getting-started/get-copy.rst:15]: I guess the following pertains more to developers. Users just need to install and run the code (e.g. the tutorials)

File: README.md (hunk 1 above)
Comment (miguelglezb) [reply README.md]: Noted.

File: docs/developer-guide/fork.rst (hunk 2 above)
Comment (miguelglezb) [reply docs/developer-guide/fork.rst]: Sorry, wrong project. Fixed.

File: docs/getting-started/get-copy.rst (hunk 3 above)
Comment (miguelglezb) [reply docs/getting-started/get-copy.rst]: I removed the reference to the developer guide so the content for users and developers is clearly separated.

Comment (miguelglezb) [reply docs/getting-started/get-copy.rst]: @paranoya can you review this change?

Comment (paranoya) [reply docs/getting-started/get-copy.rst]: Perfect :^)

File: README.md (hunk 1 above)
Comment (paranoya) [reply README.md]: OK

Comment (paranoya) [reply README.md]: Now that I think of it, perhaps we should include the activation of the virtual environment...

Comment (paranoya) [reply README.md]: Or, even better, the relevant section(s) in the documentation

Comment (miguelglezb) [reply README.md]: I'm not sure about overcrowding the README, especially if the venv information is already in the documentation

Comment (miguelglezb) [reply README.md]: I'll resolve for now, and modify README if we decided to include the venv later.

**Pull request merged**